### PR TITLE
Remove green_box_utils

### PR DESCRIPTION
**Pull request by dshiga**
As part of repo reorganization, I'd like to get rid of the green_box_utils package inside lira and distribute its parts elsewhere:

* cromwell-tools repo
for start_workflow, which is a generic function for starting workflows in Cromwell
https://github.com/broadinstitute/cromwell-tools/pull/2

* lira/utils.py
for functions that are useful only inside lira

* pipeline-tools repo
for gcs_utils, which is a set of generic functions for working with gcs and may be useful elsewhere
https://github.com/HumanCellAtlas/pipeline-tools/pull/3

I took the opportunity to reorganize things inside lira as well:
* Moved test dir inside lira module, which simplifies importing of lira components inside tests and leaves less room for error
* Created requirements.txt, which Dockerfile uses to install Python dependencies. This makes it easier to run unit tests outside the docker container (saving the time of building the image), since you can build a virtual environment from the requirements.txt sufficient to run the tests.
* Moved code out of `lira/__init__.py` and into lira/lira.py, and simplified startup of lira
* Removed unused app.yaml, previously used for deploying lira to app engine
* Renamed green_box to lira throughout

### CODE REVIEW

**Comment by rexwangcc** [comment]
Please ignore the Jenkins build, this PR will definitely cause the failure of the Jenkins build, and we are going to modify and fix the Jenkins later.

**Comment by davidbernick** [comment]
Bernick trying jenkins test again.

File: README.md [hunk 1]
```diff
@@ -17,6 +17,30 @@ Currently, the listener launches either a smartseq2 or 10x count workflow to pro
...
+This will run all of Lira's unit tests.
+
+You can also run the unit tests from inside the docker image, if you don't want to create a virtualenv.
+You can do this by running:
+```
+cd lira/test
+bash test.sh
+```
+
 ### Testing notifications locally
```
Comment (samanehsan) [inline README.md:44]: Can you update the path to `notification.json` in the example for testing notifications locally?

**Comment by samanehsan** [comment]
What do you think about moving `lira.yml` inside of the lira directory?

**Comment by samanehsan** [comment]
Did you mean to include a `requirements.txt` file in this PR?

File: lira/test/test_config.py [hunk 2]
```diff
@@ -2,72 +2,77 @@
 import unittest
 import json
 from copy import deepcopy
-from green_box_utils import listener_utils
+from lira import config
```
Comment (samanehsan) [inline lira/test/test_config.py:5]: Should this import from `utils` instead?

**Comment by samanehsan** [comment]
This looks awesome @dshiga! I just have a few comments/questions ☔️

File: lira/test/test_config.py (hunk 2 above)
Comment (dshiga) [reply lira/test/test_config.py:5]: I was in the middle of moving config classes to lira/config.py but forgot to check in all those changes, will fix.

File: README.md (hunk 1 above)
Comment (dshiga) [reply README.md:44]: Fixing

**Comment by dshiga** [comment]
Added requirements.txt and moved lira.yml as suggested. Added some missing `__init__.py` files. Moved config classes from utils.py to config.py.

**Review by samanehsan — APPROVED** [review]

**Comment by samanehsan** [comment]
Looks good to me! 🍂

**Pull request merged**